.github/workflows/ci.yaml

```
@@ -471,6 +471,149 @@ jobs:
+  #   executes those copies — it never invokes the compiler. So the run job
+  #   cannot accidentally rebuild with default flags; if the artifact is missing
+  #   it errors ("No benchmarks found") instead of measuring the wrong binary.
+  #
+  # CRITICAL — ABI match:
+  #   codspeed-macro is an ARM64 machine, so the binaries MUST be built on ARM64
+  #   too. We build on the OLDEST Ubuntu we have an ARM runner for (22.04,
+  #   glibc 2.35) so the binaries stay forward-compatible with whatever (newer
+  #   or equal) glibc the macro runner ships.
+  benchmarks-build:
+    name: "benchmarks build (baml)"
+    runs-on: blacksmith-8vcpu-ubuntu-2204-arm
```

actionlint (1.7.12)

[error] label "blacksmith-8vcpu-ubuntu-2204-arm" is unknown. available labels are "windows-latest", "windows-latest-8-cores", "windows-2025", "windows-2025-vs2026", "windows-2022", "windows-11-arm", "ubuntu-slim", "ubuntu-latest", "ubuntu-latest-4-cores", "ubuntu-latest-8-cores", "ubuntu-latest-16-cores", "ubuntu-24.04", "ubuntu-24.04-arm", "ubuntu-22.04", "ubuntu-22.04-arm", "macos-latest", "macos-latest-xlarge", "macos-latest-large", "macos-26-intel", "macos-26-xlarge", "macos-26-large", "macos-26", "macos-15-intel", "macos-15-xlarge", "macos-15-large", "macos-15", "macos-14-xlarge", "macos-14-large", "macos-14", "self-hosted", "x64", "arm", "arm64", "linux", "macos", "windows". if it is a custom label for self-hosted runner, set list of labels in actionlint.yaml config file

(runner-label)